ballot/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import patterns, url
+from django.contrib import admin
+
+from . import views
+
+admin.autodiscover()
+
+urlpatterns = patterns(
+    '',
+    url(r'^measure/(?P<measure_id>[0-9]+)$', views.measure_view),
+    url(r'^ballot/$', views.ballot_view))
```

Comment:
@adborden Right, I was confused about this--the mock-up URL doesn't take an `id`, so when I transferred I didin't accept one.

My inclination was to do this later, to make each endpoint "live" / more useful on each end. What do you think?